gobblin-utility/src/main/java/gobblin/util/concurrent/TaskSchedulerFactory.java

```diff
@@ -0,0 +1,54 @@
+package gobblin.util.concurrent;
+
+import com.google.common.base.Optional;
+import com.google.common.collect.Maps;
+import org.apache.commons.lang.StringUtils;
+
+import java.util.Map;
+
+public class TaskSchedulerFactory {
```

Comment:
Then I would rather use an enum instead of a map (also solves the issue with the static map).